cg/constants/delivery.py

```diff
@@ -207,21 +185,21 @@
         "case_tags": MUTANT_ANALYSIS_CASE_TAGS,
         "sample_tags": MUTANT_ANALYSIS_SAMPLE_TAGS,
     },
-    Workflow.RNAFUSION: {
-        "case_tags": RNAFUSION_ANALYSIS_CASE_TAGS,
-        "sample_tags": RNAFUSION_ANALYSIS_SAMPLE_TAGS,
-    },
     Workflow.RAREDISEASE: {
-        "case_tags": NF_ANALYSIS_CASE_TAGS,
-        "sample_tags": NF_ANALYSIS_SAMPLE_TAGS,
+        "case_tags": CLINICAL_DELIVERY_TAGS,
+        "sample_tags": CLINICAL_DELIVERY_TAGS,
+    },
+    Workflow.RNAFUSION: {
+        "case_tags": CLINICAL_DELIVERY_TAGS,
+        "sample_tags": CLINICAL_DELIVERY_TAGS,
     },
     Workflow.TAXPROFILER: {
-        "case_tags": NF_ANALYSIS_CASE_TAGS,
-        "sample_tags": NF_ANALYSIS_SAMPLE_TAGS,
+        "case_tags": CLINICAL_DELIVERY_TAGS,
+        "sample_tags": CLINICAL_DELIVERY_TAGS,
     },
     Workflow.TOMTE: {
-        "case_tags": NF_ANALYSIS_CASE_TAGS,
-        "sample_tags": NF_ANALYSIS_SAMPLE_TAGS,
+        "case_tags": CLINICAL_DELIVERY_TAGS,
+        "sample_tags": CLINICAL_DELIVERY_TAGS,
```

Comment:
This is quite redundant, anything we could easily do to avoid it? or would it complicate things quite a lot and better leave it for now?